packages/cashier/src/Components/Form/payment-agent-withdraw-form.jsx

```diff
@@ -182,7 +182,7 @@ const PaymentAgentWithdrawForm = ({
             currency,
             payment_agent:
                 values.payment_method === 'payment_agent'
-                    ? {}
+                    ? payment_agent_list.find(pa => pa.value === values.payment_agent)
```

Comment:
Could you rewrite this line in a way to refuse code duplication?
something like this may work: `payment_agent: payment_agent_list.find(pa => pa.value === values[values.payment_method]),`